__tests__/app.test.js

```diff
@@ -32,28 +33,28 @@ describe("Task-3 GET:/api/topics", () => {
       .get("/api/topics")
       .then((res) => {
         const topics = res.body.topics;
-        topics.forEach(topic => {
-          expect(topic).toHaveProperty("description",expect.any(String));
-          expect(topic).toHaveProperty("slug",expect.any(String));
+        topics.forEach((topic) => {
```

Comment:
Make sure you protect this `.forEach` with a length condition. If the array returned is unpopulated, this test will give a false positive result.